src/lib/operators/useTestCase.ts

```diff
@@ -62,19 +62,30 @@ export function useTestCase(
         const tc = p?.testCases.find((item) => item.id === caseId);
         if (!tc) return;
 
+        const t = tc?.tests.find((item) => item.id === testId);
+        const s = t?.steps.find((item) => item.id === stepId);
```

Comment:
For readability please let's avoid using single letter variables. This is hugely important in every project, even more given this is opensourced.

`p` should be `project`
`tc` should be `testCase`
`t` should be `test`
`s` should be `step`

Reply:
This should be applied in all vars created in `useTestCase.ts` `useProject.ts` `useProjects.ts` `useTest.ts` `useStep.ts`, maybe a new dedicated issue should be created.